view/search/index/search.phtml

```diff
@@ -44,6 +44,13 @@
 <div id="search-form-wrapper">
     <?php echo $this->searchForm(); ?>
 </div>
+<?php if(!empty($queryString)) : ?>
+    <div class="search-results__search-summary">
+        <?php echo $this->partial('search/search_summary', [
+            'queryString' => $queryString,
```

Comment:
It looks like you are already using the term 'summary' so you should probably use that in the interface and method names (see previous comment on the interface)

Reply:
All partial names in omeka and this module use `-` instead of `_`. Better to follow the existing convention